bin/next-remote-watch

```diff
@@ -21,8 +27,25 @@ const handle = app.getRequestHandler()
 app.prepare().then(() => {
   // if directories are provided, watch them for changes and trigger reload
   if (program.args.length > 0) {
-    chokidar.watch(program.args).on('change', () => {
+    chokidar.watch(program.args).on(program.event, async (path, event = defaultWatchEvent) => {
       app.hotReloader.send('building')
+
+      if (program.command) {
+        try {
+          const { stdout, stderr } = await exec(`${program.command} ${path} ${event}`)
+          if (stderr) {
+            throw stderr
+          }
+
+          console.log(stdout)
+
+        } catch (e) {
+          console.error('Remote command failed')
+          console.error(e)
+          return e
+        }
+      }
```

Comment:
i think we need to chain our `command` execution to the `reloadPage` right? bc otherwise if that hangs (say you forget a return in your `command` script) then you'd still get the `reloadPage` event. 

i'm not totally confident so check me on this.

Reply:
It's inside a try/catch block so we're clear on this 😀

Reply:
wouldn't that only protect us if it errors and not if it hangs (or just generally doesn't return something to `stderr`) ? and wouldn't it also not guarantee the next logs wouldn't be like 

```
[ wait ]  compiling ...
[ ready ] compiled successfully
Remote command failed
```

...which would be sort of confusing behavior. I would expect to see...

```
[ wait ]  compiling ...
Remote command failed
```

Reply:
if it hangs i think we're in trouble regardless, but i see what you're saying -- you want the reload not to happen unless the command is successful? technically i don't see any reason to make the two dependent though -- we could have the site compile successfully and the remote command fail right?

Reply:
yeah, i'm not 100% sure how i think it should be, tbh. i can go either way.

right now, i'm just thinking about learn authors who will be completely blind to the idea that there's a synchronization script happening and will be confused when the log says `compiled successfully` but the content isn't there. 

anyways, we'll use it as-is and see how it feels. i just wanted to raise it for our consideration.